### PR TITLE
Slik.Subversion version 1.14.2001

### PR DESCRIPTION
**Pull request by rhuijben**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/59580)

### CODE REVIEW

**Comment by rhuijben** [comment]
Looks like a transient failure
`Exception Message: Cannot get DNS address for host 'winget-pme.azurefd.net' (No such host is known). (type TeamFoundationServerException)`

**Comment by rhuijben** [comment]
Pushed exactly the same commit with a new timestamp, to retry validation.

File: manifests/s/Slik/Subversion/1.14.2001/Slik.Subversion.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Slik.Subversion
+PackageVersion: 1.14.2001
+PackageLocale: en-US
+Publisher: Slik and the SharpSvn Project
+PackageName: Subversion 1.14.2
+License: Apache-2
+ShortDescription: Subversion Client
```
Comment (Trenly) [inline manifests/s/Slik/Subversion/1.14.2001/Slik.Subversion.locale.en-US.yaml:10]: Please provide a better description

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/s/Slik/Subversion/1.14.2001/Slik.Subversion.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Slik.Subversion
+PackageVersion: 1.14.2001
+PackageLocale: en-US
+Publisher: SlikSvn & The SharpSvn Project
+PackageName: Subversion 1.14.2
```
Comment (ItzLevvie) [inline manifests/s/Slik/Subversion/1.14.2001/Slik.Subversion.locale.en-US.yaml:8]:
```suggestion
PackageName: Slik Subversion
```
to match Control Panel.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/s/Slik/Subversion/1.14.2001/Slik.Subversion.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Slik.Subversion
+PackageVersion: 1.14.2001
+PackageLocale: en-US
+Publisher: SlikSvn & The SharpSvn Project
+PackageName: Subversion 1.14.2
+License: Apache-2
```
Comment (ItzLevvie) [inline manifests/s/Slik/Subversion/1.14.2001/Slik.Subversion.locale.en-US.yaml:9]:
```suggestion
License: Apache-2.0
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**